django/db/backends/postgresql/schema.py

```diff
@@ -11,8 +11,10 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     sql_set_sequence_max = "SELECT setval('%(sequence)s', MAX(%(column)s)) FROM %(table)s"
     sql_set_sequence_owner = 'ALTER SEQUENCE %(sequence)s OWNED BY %(table)s.%(column)s'
 
-    sql_create_index = "CREATE INDEX %(name)s ON %(table)s%(using)s (%(columns)s)%(extra)s%(condition)s"
-    sql_delete_index = "DROP INDEX IF EXISTS %(name)s"
+    sql_create_index = (
+        "CREATE %(index)s %(name)s ON %(table)s%(using)s (%(columns)s)%(extra)s%(condition)s"
+    )
+    sql_delete_index = "DROP %(index)s IF EXISTS %(name)s"
```

Comment:
I would leave `INDEX` and add only `%(concurrent)`.

Reply:
@charettes had a suggestion to introduce another template. I've used that.